### PR TITLE
creation routage backend

### PR DESCRIPTION
**Pull request by n-faycel**

### CODE REVIEW

**Review by ThomasDCW — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: backend/src/models/LessonManager.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const AbstractManager = require("./AbstractManager");
+
+class LessonManager extends AbstractManager {
+  static table = "lesson";
+
+  findAll() {
+    return this.connection.query(`select * from  ${LessonManager.table}`);
```
Comment (LoicBrassart) [inline backend/src/models/LessonManager.js:7]: No need, it's present in the AbstractManager

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A bit of code to remove

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A bit of code to remove

**Review by n-faycel — COMMENTED** [review]
ok

**Review by ThomasDCW — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**